flink-ai-flow/bin/start_aiflow.py

```diff
@@ -24,21 +24,32 @@
 
 
 def create_default_sever_config(root_dir_path):
+    airflow_deploy_path = os.environ["AIRFLOW_DEPLOY_PATH"]
```

Comment:
It's recommented to use argment better than the environment variable.

Reply:
Okay.